Makefile

```diff
@@ -64,7 +64,7 @@ new_env: clean
 	then\
 		pipenv --rm;\
 		pipenv --python 3.8;\
-		pipenv install --dev --skip-lock --clear;\
+		pipenv install --dev --skip-lock;\
```

Comment:
```
--clear                         Clears caches (pipenv, pip, and pip-tools).
                                  [env var: PIPENV_CLEAR]
```
Is there a similar flag for linux version of pipenv?

Reply:
I wasn't aware that --clear was OS-specific. Let me check.

Reply:
`pipenv lock --clear` seems to work. The problem arises only within `pipenv install --dev --skip-lock --clear`.

Reply:
From the manpage:

```
   $ pipenv install
       $ pipenv install is used for installing packages into the pipenv virtual environment and updating your Pipfile.

       Along with the basic install command, which takes the form:

          $ pipenv install [package names]

       The user can provide these additional parameters:

          • --two — Performs the installation in a virtualenv using the system python2 link.

          • --three — Performs the installation in a virtualenv using the system python3 link.

          • --python — Performs the installation in a virtualenv using the provided Python interpreter.

          WARNING:
              None of the above commands should be used together. They are also destructive and will delete your current virtualenv before replacing it with an appropriately versioned one.

          NOTE:
              The virtualenv created by Pipenv may be different from what you were expecting.  Dangerous characters (i.e. $`!*@" as well as space, line feed, carriage return, and tab) are converted to underscores. Additionally, the
              full path to the current folder is encoded into a "slug value" and appended to ensure the virtualenv name is unique.

          • --dev — Install both develop and default packages from Pipfile.lock.

          • --system — Use the system pip command rather than the one from your virtualenv.

          • --ignore-pipfile — Ignore the Pipfile and install from the Pipfile.lock.

          • --skip-lock — Ignore the Pipfile.lock and install from the Pipfile. In addition, do not write out a Pipfile.lock reflecting changes to the Pipfile.

```

It seems there's no option.

Reply:
ok